pwndbg/gdblib/abi.py

```diff
@@ -1,18 +1,19 @@
 from __future__ import annotations
 
 import functools
-from typing import Any
 from typing import Callable
-from typing import Optional
 from typing import TypeVar
 
 import gdb
+from typing_extensions import ParamSpec
 
 import pwndbg.color.message as M
 
-T = TypeVar("T")
+_P = ParamSpec("_P")
+_D = TypeVar("_D")
+_T = TypeVar("_T")
```

Comment:
nit: I see some places we use `T` and some places we use `_T`,  let's be consistent. I personally prefer `T`, and I think that's more common in the rest of the code.